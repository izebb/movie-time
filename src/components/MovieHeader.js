import React from 'react';
import styled from 'styled-components';
import logo from './header-logo-large.png';

const Header = styled.div`
  background: #1e282d;
  padding: 20px 40px;
  text-align: center;
  margin-bottom: 50px;
  position: sticky;
  width: 100%;
  left: 0;
  top: 0;
  z-index: 2;
`;

const Image = styled.img`
  width: 140px;
`;

export const MovieHeader = () => {
  return (
    <Header>
      <Image src={logo} alt="" />
    </Header>
  );
};
