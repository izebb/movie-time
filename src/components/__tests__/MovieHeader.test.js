import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { MovieHeader } from '../MovieHeader';

describe('MovieCard', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<MovieHeader />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
