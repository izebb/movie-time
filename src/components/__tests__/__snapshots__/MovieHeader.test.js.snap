// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard should render without crashing 1`] = `
<MovieHeader__Header>
  <MovieHeader__Image
    alt=""
    src={
      Object {
        "process": [Function],
      }
    }
  />
</MovieHeader__Header>
`;
